### ISSUE TITLE
Ice in the sky

### ISSUE THREAD

**Issue by Starkitobonito** [PaperMC/Paper#5941]
### Timings or Profile link

https://www.spigotmc.org/members/starkito.968222/

### Description of issue

The ice shows in the sky
![javaw_2021-06-21_16-24-52](https://user-images.githubusercontent.com/68711129/122830811-7be9e280-d2ae-11eb-850e-4000b173b85b.png)
![javaw_2021-06-21_16-26-24](https://user-images.githubusercontent.com/68711129/122830816-7db3a600-d2ae-11eb-8c5c-e743c743d547.png)

### Plugin list

-Skinresorter

### Server config files

```
Paste configs or paste.gg link here!
```
[Files.zip](https://github.com/PaperMC/Paper/files/6689629/Files.zip)

### Agreements

- [X] I am running the latest version of Paper available from https://papermc.io/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Paper.

**Comment by Prof-Bloodstone** [comment]
You opened wrong issue type - it's not a performance issue but a bug. 
Anyway, did you run one of the first builds on your server, which were only posted in 1.17 channel on Discord? Or did you only use ones from website?
Were these chunks visited earlier? What datapacks do you use?
Also, please update the issue with your Paper version.

**Comment by DenWav** [comment]
Please provide world seed and coordinates.

**Comment by Starkitobonito** [comment]
This all are my datapacks, one is for caves generation but i never seen this before in a singleplayer world.
![chrome_2021-06-21_16-50-42](https://user-images.githubusercontent.com/68711129/122832484-029fbf00-d2b1-11eb-93e1-872abf1bf411.png)

I am using the experimental version, which comes out in Apex Hosting, they told me in support that it was the last one.
![chrome_2021-06-21_16-53-49](https://user-images.githubusercontent.com/68711129/122832820-88236f00-d2b1-11eb-9949-bee568130b52.png)

Im generating all this new world, im testing some things in creative and found this
![javaw_2021-06-21_16-56-43](https://user-images.githubusercontent.com/68711129/122832889-ab4e1e80-d2b1-11eb-884d-3b5d97d0ceef.png)
There are the cords and the seed of ther world is: tortura

**Comment by Prof-Bloodstone** [comment]
> I am using the experimental version, which comes out in Apex Hosting, they told me in support that it was the last one.

This tells us nothing. What is the output of `/version`? Did you run it on older versions too?

**Comment by Starkitobonito** [comment]
Never mind
![javaw_2021-06-21_17-00-26](https://user-images.githubusercontent.com/68711129/122833215-316a6500-d2b2-11eb-9657-7c37aa79972c.png)


I was testing in a singleplayer world, and I found the same problem, probably a data pack error. Sorry for the mistake, I saw a similar problem on the discord channel with some blocks and I thought it could be a paper problem. Keep up with the great work!
